pkg/controllers/managementuser/rbac/namespace_handler.go

```diff
@@ -87,6 +88,30 @@ func (n *nsLifecycle) Remove(obj *v1.Namespace) (runtime.Object, error) {
 }
 
 func (n *nsLifecycle) syncNS(obj *v1.Namespace) (bool, error) {
+	// add fleet namespace to system project
+	if IsFleetNamespace(obj) &&
```

Comment:
I would be ok with this, but we are actually backporting https://github.com/rancher/rancher/issues/31209 to 2.5 then we should open a new issue for 2.5 and let qa verify it.

Reply:
Removed.